### DISCUSSION TITLE
Why is the model file location relevant?

### DISCUSSION THREAD

**Discussion by strakehyr** [timeseriesAI/tsai#525] [Q&A]
Just wanted to know why models can't be moved of place, and their location is fixed once exported/saved. Are there any plans to tackle this limitation or is there a reason for it?

**Comment by oguiza** [comment]
> why models can't be moved of place, and their location is fixed once exported/saved.

What makes you think that? You can use your exported learner wherever you want.

**Reply by strakehyr** [reply]
Haven't tried it recently, but isn't it location-bound?

**Reply by oguiza** [reply]
No, it isn't.
